pokedex/src/Components/Pokeinfo/Pokeinfo.jsx

```diff
@@ -0,0 +1,41 @@
+import React from "react";
+import "../Pokeinfo/Pokeinfo.css";
+
+const Pokeinfo = ({ data }) => {
+  return (
+    <>
+      {!data ? (
+        ""
+      ) : (
+        <>
+          <h1>{data.name}</h1>
+          <img
+            src={`https://raw.githubusercontent.com/PokeAPI/sprites/master/sprites/pokemon/other/dream-world/${data.id}.svg`}
+            alt=""
```

Comment:
`alt=""
`

Every `img` tag must have a **descriptive** `alt` attribute (the **name** of the image is usually there) to show the **alternative** text if the image is not uploaded, and it’s needed for **screen readers**